src/components/ui/Badge/Badge.module.scss

```diff
@@ -0,0 +1,30 @@
+.badge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--color-orange);
+  text-transform: uppercase;
+  font-weight: bold;
+  width: fit-content;
+  border-radius: 8px;
```

Comment:
I'm wondering if it would be beneficial for us to extract that 8px as some variable indicating default `border-radius`. We may want to sue it in schedule, FAQ and so on, wdyt?

Reply:
Good idea 👍